src/Nethermind/Nethermind.Synchronization/Peers/SyncPeerPool.cs

```diff
@@ -113,7 +113,7 @@ public void ReportBreachOfProtocol(PeerInfo? peerInfo, string details)
             if (peerInfo is not null)
             {
                 _stats.ReportSyncEvent(peerInfo.SyncPeer.Node, NodeStatsEventType.SyncFailed);
-                peerInfo.SyncPeer.Disconnect(DisconnectReason.BreachOfProtocol, details);
+                peerInfo.SyncPeer.Disconnect(InitiateDisconnectReason.SyncPeerPoolBreachOfProtocol, details);
```

Comment:
I think it would be helpful to know what exactly that breach of protocol was, would require changing the incoming method.

Reply:
Done